client/src/features/project/components/migrations/MigrationHelpers.tsx

```diff
@@ -226,7 +227,7 @@ export function DetailsSection({
   if (titleInfo) {
     const externalLinkStyles = {
       className: "text-rk-white",
-      role: "text",
+      role: "text" as ExternalLinkRole,
```

Comment:
nitpick: using `as const` for string literals works just as well so you don't have to import `ExternalLinkRole`.

```suggestion
      role: "text" as const,
```